### ISSUE TITLE
Python 3.13 support

### ISSUE THREAD

**Issue by Mogost** [profusion/sgqlc#247]
https://github.com/profusion/sgqlc/blob/d36984ac71165db6437c63b5aafd69b05eab52b9/pyproject.toml#L62-L63

Those lines block installation on Python 3.13 which is released.

Current possible workaround is downgrading to version 16.3

**Comment by barbieri** [comment]
yeah, those were because of a dep, I'll see if the dep is already fixed and will relax that one

**Comment by barbieri** [comment]
yeah, still a blocker: https://pypi.org/project/coveralls/

```
The current project's supported Python range (>=3.8) is not compatible with some of the required packages Python requirement:
  - coveralls requires Python <3.13,>=3.8, so it will not be satisfied for Python >=3.13
  - coveralls requires Python <3.13,>=3.8, so it will not be satisfied for Python >=3.13

Because no versions of coveralls match >4,<4.0.1 || >4.0.1
 and coveralls (4.0.0) requires Python <3.13,>=3.8, coveralls is forbidden.
So, because coveralls (4.0.1) requires Python <3.13,>=3.8
 and sgqlc depends on coveralls (>=4), version solving failed.
```

**Comment by barbieri** [comment]
https://github.com/TheKevJames/coveralls-python/issues/549

**Comment by zodiacfireworks** [comment]
> [TheKevJames/coveralls-python#549](https://github.com/TheKevJames/coveralls-python/issues/549)

@barbieri, maybe we can get rid of that dependency if we use the [universal reporter](https://github.com/coverallsapp/coverage-reporter) and remove the coveralls package from the dependencies.

**Comment by barbieri** [comment]
hum, I'll check the universal reporters and let you know, thanks @zodiacfireworks.

@wyardley yes, coveralls is just build time

**Comment by barbieri** [comment]
**FINALLY** done, thanks you all, in particular @zodiacfireworks for pointing me to the new GH action that does the reporting on its own image/container.

**Comment by wyardley** [comment]
Awesome. Thanks so much @barbieri!

**Comment by wyardley** [comment]
@barbieri could a release get cut? I think 16.4 from last Sept is still the latest available via PyPi?

**Comment by barbieri** [comment]
sorry about the delay, I was traveling. Here it is https://pypi.org/project/sgqlc/16.5/
